src/lib/pubkey/ec_group/ec_inner_data.cpp

```diff
@@ -61,6 +61,13 @@ EC_Group_Data::EC_Group_Data(const BigInt& p,
       }
    }
 
+   // Try a generic pcurves instance
+   if(!m_pcurve && !m_has_cofactor) {
+      m_pcurve = PCurve::PrimeOrderCurve::from_params(p, a, b, g_x, g_y, order);
+      // possibly still null here, if parameters unsuitable or if the
+      // pcurves_generic module wasn't included in the build
+   }
```

Comment:
Out of caution: Don't forget to consider https://github.com/randombit/botan/pull/4630

Reply:
Rebased and added setting the type correctly